src/OidcClient.php

```diff
@@ -162,6 +162,23 @@ public function generateAuthorizationRedirect(
     return new RedirectResponse(sprintf('%s%s%s', $this->getAuthorizationEndpoint(), $endpointHasQuery ? '&' : '?', http_build_query($data)));
   }
 
+  /** {@inheritDoc} */
+  public function generateEndSessionEndpointRedirect(
+      OidcTokens $tokens,
+      string $postLogoutRedirectUrl,
+      array $additionalQueryParams = []
+  ): RedirectResponse {
+    $data = array_merge($additionalQueryParams, [
+        'client_id'                => $this->clientId,
+        'id_token_hint'            => $tokens->getIdToken(),
+        'post_logout_redirect_uri' => $postLogoutRedirectUrl,
```

Comment:
I assume this URI is not required, so we should probably support that here as well in case someone want to do this manually.

Reply:
I will check, but with one of the providers I tested against it displayed an error when the uri wasn't there. I now have a Keycloak backend as test provider and will check what happens there after redirecting to there logout endpoint.

Reply:
I added a config setting to disable the automatic redirect based on the logout_target, by default I enabled this one. I figures you want your user to redirect back to your application in 99% of the time and not remain on the logout confirmation of your identity provider.

Reply:
👍🏻